### PR TITLE
ethereum/scripts: reduce register script

### LINKED ISSUE #2224: Tilt Eth registrations should dynamically read env vars
agreed the old scripts should be deleted. 
additionally, we should consider dynamically picking up the VAAs from the environment 
so adding new chains does not require updating all the deployment scripts. 
This is done for aptos as an example: https://github.com/wormhole-foundation/wormhole/blob/main/aptos/scripts/register_devnet#L13

and I think the other chains too

_Originally posted by @kcsongor in https://github.com/wormhole-foundation/wormhole/issues/2214#issuecomment-1377248236_

### PR DESCRIPTION
**Pull request by panoel**
Fixes #2224

### CODE REVIEW

**Review by evan-gray — APPROVED** [review]
nice! 🙌

File: ethereum/scripts/register_all_chains.js [hunk 1]
```diff
@@ -6,16 +6,19 @@ const NFTBridge = artifacts.require("NFTBridgeEntrypoint");
...
-const wormchainTokenBridgeVAA = process.env.REGISTER_WORMCHAIN_TOKEN_BRIDGE_VAA;
-const aptosTokenBridgeVAA = process.env.REGISTER_APTOS_TOKEN_BRIDGE_VAA;
+
+// The input parameter is a RegExp
+// It returns an array of process.env variables satisfying the input RegExp
+function getFilteredEnvs(regexp) {
+  const filteredEnvs = [];
+  for (const [key, value] of Object.entries(process.env)) {
+    if (regexp.test(key) && value) {
+      console.log("getFilteredEnvs: pushing " + key);
```
Comment (kcsongor) [inline ethereum/scripts/register_all_chains.js:16]: I feel like we don't need this log here

**Review by kcsongor — APPROVED** [review]
left a minor comment, not a blocker 👍

File: ethereum/scripts/register_all_chains.js (hunk 1 above)
Comment (evan-gray) [reply ethereum/scripts/register_all_chains.js:16]: Could go either way, was kinda nice to see which bridges were getting registered (though maybe change the log output). I’m 🤷‍♂️. Need this for tests though so I’ll merge and we can revisit in the next “make CI have less output” event. 🙃

**Review by evan-gray — APPROVED** [review]

File: ethereum/scripts/register_all_chains.js (hunk 1 above)
Comment (kcsongor) [reply ethereum/scripts/register_all_chains.js:16]: agreed it's nice to see which chain is getting registered, but the log shouldn't be emitted by the helper function that grabs the env var

**Pull request merged**